blazeutils/spreadsheets.py

```diff
@@ -26,6 +31,11 @@
 from .decorators import deprecate
 
 
+def _openpyxl_required():
+    if openpyxl is None:
+        raise ImportError('openpyxl library is required to use this function or class')
```

Comment:
Would this ever get called? We choose the implementation based on the presence of the library. So we shouldn't ever choose openpyxl classes unless openpyxl is installed?